packages/fxa-settings/src/pages/Signin/SigninConfirmed/index.test.tsx

```diff
@@ -54,11 +53,9 @@ describe('SigninConfirmed', () => {
 
     fireEvent.click(passwordResetContinueButton);
     expect(logViewEvent).toHaveBeenCalledWith(
-      'signin-confirmed',
-      'signin-confirmed.continue',
-      {
-        entrypoint_variation: 'react',
-      }
+      viewName,
+      `${viewName}.continue`,
+      REACT_ENTRYPOINT
```

Comment:
@millsoper @vpomerleau @vbudhram I think there's a few of these that might be incorrect, we may want to create a `logFlowEvent` function, this is `logViewEvent`:

```
export function logViewEvent(
  viewName: string,
  eventName: string,
  eventProperties: Hash<any> = {}
) {
  if (viewNamePrefix) {
    viewName = `${viewNamePrefix}.${viewName}`;
  }

  logEvents([`${viewName}.${eventName}`], eventProperties);
}
```

I corrected some in `cannot_create_account` and `cookies_disabled` to prepend the word `flow`; I think these should be ```'flow', `${viewName}.continue`, REACT_ENTRYPOINT``` but that seems funky given the param names. Maybe we can take a look as we build out components or make a note on the functional ticket so this doesn't scope creep too much, I've got a whole other PR to do to close this ticket. 😆

Reply:
`viewNamePrefix` 😢, I remember when this was introduced. It was so that we could distinguish users that signup from oauth and signup from signin. It became a pain to manage and made it very hard to know what event a page would emit what metrics.

I think it is fine to update these as needed or in the functional tickets. Ideally would love to rethink these metrics but that is much bigger issue.

Reply:
> It became a pain to manage and made it very hard to know what event a page would emit what metrics.

Argh yeah, it's still a pain; I totally agree about wanting to rethink all these. We have that "understanding FxA users journey" goal and I'm hoping we can rethink these at that point.